pkg/cmd/delete/edge_storage/bucket.go

```diff
@@ -46,20 +50,53 @@ func (b *bucket) runE(cmd *cobra.Command, _ []string) error {
 	ctx := context.Background()
 	err := client.DeleteBucket(ctx, b.name)
 	if err != nil {
+		if msg.ERROR_NO_EMPTY_BUCKET == err.Error() {
+			if !b.force {
+				if !utils.Confirm(b.factory.GlobalFlagAll, msg.ASK_NOT_EMPTY_BUCKET, false) {
+					return nil
+				}
+			}
+			logger.Info("Delete all objects bucket", zap.Any("bucket-name", b.name))
+			if err := deleteAllObjects(client, ctx, b.name, ""); err != nil {
+				return err
+			}
+			err := client.DeleteBucket(ctx, b.name)
+			if msg.ERROR_NO_EMPTY_BUCKET == err.Error() {
```

Comment:
Isn't better to use "errors.Is" to compare errors?

Reply:
maybe, in that case it's not necessary!